### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by tdincer**

### CODE REVIEW

File: docs/src/citation.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Citation
+
+If your work uses this Element, please cite the following manuscript and Research
+Resource Identifier (RRID).
+
++ Yatsenko D, Nguyen T, Shen S, Gunalan K, Turner CA, Guzman R, Sasaki M, Sitonic D,
+  Reimer J, Walker EY, Tolias AS. DataJoint Elements: Data Workflows for
+  Neurophysiology. bioRxiv. 2021 Jan 1. doi: https://doi.org/10.1101/2021.03.30.437358
+
++ DataJoint Elements ([RRID:SCR_021894](https://scicrunch.org/resolver/SCR_021894)) -
```
Comment (CBroz1) [inline docs/src/citation.md:10]: Please add [automated versioning to the citation page](https://github.com/datajoint/element-deeplabcut/commit/352c44f98325cb79a249026d84310bff8417e7e2)

**Comment by kabilar** [comment]
@tdincer Is this pull request ready for review?

File: docs/mkdocs.yaml [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+docs_dir: ./src
+theme:
+  font:
+    text: Roboto Slab
+    code: Source Code Pro
+  name: material
+  custom_dir: src/.overrides
+  icon:
+    logo: main/project-logo-black
+  favicon: assets/images/project-logo-color.png
```
Comment (CBroz1) [inline docs/mkdocs.yaml:72]: It looks like these are the old versions

**Comment by kabilar** [comment]
Superseded by #18.

**Pull request closed without merging**